src/main/openapi/ckan.yaml

```diff
@@ -267,8 +263,15 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/CkanValueLabel"
+        contact_name:
```

Comment:
@Markus92 why has this changed back to a single valued contact point?

Reply:
The DCAT metadata standard that we'd be using will only allow for a single value for contact point, see also https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-gdi-userportal/pull/89

On the interface side I think multi-valued could / should still be supported, so that's why output would be a list with a single value on it.

Reply:
In consultation with my colleagues, reverted this change so it's only the creator mapping now.